DECLARING_DEPENDENCIES.md

```diff
@@ -0,0 +1,209 @@
+# Declaring Dependencies on Google Java libraries
+
+Google maintains a number of open source Java libraries that make it
+easier to use services in Google Cloud Platform. Additionally Google
+maintains several foundational libraries that can be used for very
+general purposes, which the GCP libraries also depend on. A user of
+these libraries needs to use compatible versions of them in order to
+avoid dependency conflicts. This document explains how to use of BOMs
+(Bills of Materials) to accomplish that, and additional practices that
```

Comment:
Removing the comma creates a syntactic ambiguity (it could mean: "to accomplish that and [accomplish] additional practices" but the correct reading is "This document explains how to ... to accomplish that, and [explains] additional practices". I'll add the word "explains" explicitly.